validator/engine/validator.js

```diff
@@ -5932,6 +5932,10 @@ amp.validator.ValidationResult.prototype.outputToTerminal = function(
   const {status} = this;
   if (status === amp.validator.ValidationResult.Status.PASS) {
     terminal.info('AMP validation successful.');
+    terminal.info('Review our \'Publishing checklist\' to ensure '
+        + 'successful AMP document distribution. '
+        + 'https://amp.dev/documentation/guides-and-tutorials/'
```

Comment:
have we considered a shortener url now that amp.dev has such long urls?

Reply:
I don't think we use a particular one, but I've been told bit.ly is okay

Reply:
done.